cmd/eskip/load.go

```diff
@@ -103,9 +103,26 @@ func checkCmd(a cmdArgs) error {
 		return err
 	}
 
+	err = checkEmptyBackends(routes)
+
+	if err != nil {
+		return err
+	}
+
 	return checkRepeatedRouteIds(routes)
 }
 
+func checkEmptyBackends(routes []*eskip.Route) error {
+
+	for _, route := range routes {
+		if route.BackendType == eskip.NetworkBackend && route.Backend == "" {
+			return errors.New("Route has empty backend: " + route.Id)
```

Comment:
fmt.Errorf would be cleaner

Reply:
Error message should start from lower case (I know you've copied this from checkRepeatedRouteIds)

Reply:
Do you want me to also fix the ones in `checkRepeatedRouteIds`? To me, it makes little sense to have different ways of formatting errors few lines apart...

Reply:
Yes, please fix another case